src/NuGet.Indexing/AzureDirectorySynchronizer.cs

```diff
@@ -104,7 +104,7 @@ private static void UnidirectionalSync(AzureDirectory sourceDirectory, Directory
 
             // we'll remove old files from both AzureDirectory's cache directory, as well as our destination directory
             // (only when older than 45 minutes - old files may still have active searches on them so we need a margin)
-            var referenceTimestamp = LuceneTimestampFromDateTime(DateTimeOffset.UtcNow.AddMinutes(-45));
+            var referenceTimestamp = LuceneTimestampFromDateTime(DateTime.UtcNow.AddMinutes(-45));
```

Comment:
There is not currently any unit testing around this code.
Adding some would be a nontrivial effort as we don't currently have any framework to load/unload a consistent set of files.
The other stuff used here (ie the Directory implementations) are not our implementations.
I guess we could add testing around those, but I struggle to see the value add there.